config/application.rb

```diff
@@ -44,6 +44,10 @@ class Application < Rails::Application
     config.autoload_paths += Dir["#{config.root}/lib"]
     config.eager_load_paths += Dir["#{config.root}/lib"]
 
+    Dir["./app/middlewares/*.rb"].sort.each do |file|
```

Comment:
Do we really need to require these explicitly? Generally, subdirectories of `app/` are autoloaded, but if they aren't yet at this point, so a comment would maybe help.

Reply:
Added a comment